<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>150</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>create user form</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>enter username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>190</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>enter password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>180</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="create_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create user</string>
   </property>
  </widget>
  <widget class="QLabel" name="messages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>240</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Ubuntu&quot;;color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gotoconfig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Go To Config Screen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chart">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>charts</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
